charts/stable/tt-rss/values.yaml

```diff
@@ -33,13 +36,19 @@ env:
   # @default -- postgresql.postgresqlDatabase value
   TTRSS_DB_NAME: ""
 
+# -- Configure the services for the chart here.
 service:
-  port:
-    port: 8080
+  main:
+    ports:
+      http:
+        port: 8080
 
+# -- Configure the ingresses for the chart here.
 ingress:
-  enabled: false
+  main:
+    enabled: false
 
+# -- Configure the probes for the chart here.
 probes:
   startup:
     enabled: true
```

Comment:
```suggestion
```

We can probably leave it up to the user if they want to use probes or not for this chart :)